### DISCUSSION TITLE
misc/update-lxcs.sh: call custom /update.sh script if available

### DISCUSSION THREAD

**Discussion by tzahari** [tteck/Proxmox#2395] [Ideas]
Some containers need different updates (e.g. via pip or git) and therefore it would be nice to have the possibility to put a shell script into the container (e.g. `/update.sh`) and this gets called by the `update-lxcs.sh` after the normal update command.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/1777

**Reply by tzahari** [reply]
Missed that, sorry.
